submissions/opeyemiPeter/database/transportation-company.sql

```diff
@@ -6,42 +6,79 @@ SHOW DATABASES;
- age VARCHAR(4),
+ age VARCHAR(4) ,
  PRIMARY KEY(id)
 );
-SELECT * FROM registeredPassengers; 
-
-
+SELECT * FROM registered_passengers;
+select COUNT(*) from registered_passengers;
+DROP table registered_passengers;
 -- Creating the Passenger trip details table
-CREATE TABLE tripDetails (
+CREATE TABLE trip_details (
  id INT NOT NULL AUTO_INCREMENT,
- passengerId INT NOT NULL,
- passengerClass INT NOT NULL,
- passengerTicketNo VARCHAR(20) NOT NULL,
- tripFare DECIMAL(19,4) NOT NULL,
+ passenger_id INT NOT NULL,
+ passenger_class INT NOT NULL ,
+ passenger_ticket_no VARCHAR(50) NOT NULL,
+ trip_fare DECIMAL(19,4) NOT NULL,
  cabin VARCHAR(20),
- numberOfParentsChildren INT NOT NULL DEFAULT  0,
- numberOfSiblingsSpouse INT NOT NULL DEFAULT 0,
- embarkationPoint VARCHAR(255) NOT NULL,
+ number_of_parents_Children INT NOT NULL                    ,
+ number_0f_siblings_spouse    INT NOT NULL,
+ embarkation_point VARCHAR(10) NOT NULL,
  PRIMARY KEY(id),
- FOREIGN KEY(passengerId) REFERENCES registeredPassengers(id)
+ FOREIGN KEY(passenger_id) REFERENCES registered_passengers(id)
 );
-
-SELECT * FROM tripDetails;
+SELECT * FROM trip_details;
+DROP table trip_details;
+select COUNT(*) from trip_details;
 
 -- Creating Accident case table
-CREATE TABLE accidentCases(
+CREATE TABLE accident_cases(
  id int not null auto_increment,
- passengerId int not null,
- accidentStatus tinyint not null,
+ passenger_id int not null,
+ accident_status tinyint not null,
  primary key(id),
- foreign key(passengerId) references registeredPassengers(id)
+ foreign key(passenger_id) references registered_passengers(id)
 );
+Select * from accident_cases;
+select COUNT(*) from accident_cases;
+drop tables accident_cases;
+-- Task 3 Transportation Data Query.
+
+# 1. Total number of passengers who survived
+	select count(*) from accident_cases where accident_status > 0;
+	-- Answers => 342
 
-SELECT * FROM accidentCases;
+# 2. total number of unsurvived passengers
+  SELECT COUNT(*) FROM accident_cases WHERE accident_status = 0;
+  -- Answers => 549
 
+# 3. Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+	SELECT full_name, sex from registered_passengers
+	inner join trip_details on 
+    registered_passengers.id = trip_details.passenger_id 
+    where (age  < '27') and (embarkation_point = 
+    ( "Q%"| "C%"));
+    
+# 4. Number of passengers who embarked at Southampton and survived
+	SELECT count(*) from accident_cases left join trip_details on 
+    accident_cases.passenger_id = trip_details.passenger_id where 
+    accident_status > 0 and embarkation_point = 'S';
+    -- Answers => 218;
 
+    
+-- 5 total number of passengers who paid a fare greater than $100 and 
+-- above the age of 35 with siblings or spouses on board
+# 5. SELECT COUNT(*) FROM registered_passengers 
+    inner join trip_details on registered_passengers.id = trip_details.id
+    where trip_fare > 100 and age > '35' and number_0f_siblings_spouse > 0; 
```

Comment:
What are you doing from line 72 - 76

Reply:
I am getting the total number of passengers who paid a fare greater than $100 and above the age of 35 with siblings or spouses on board

Reply:
I am getting the total number of passengers who paid a fare greater than $100 and above the age of 35 with siblings or spouses on board

Reply:
I am getting the total number of passengers who paid a fare greater than $100 and above the age of 35 with siblings or spouses on board 
@adaugochi 

then next line was get the id, name and the total number of passengers who paid a fare greater than $100 and above the age of 35 with siblings or spouses on board.

Reply:
Your code started from line 75. Also, you are doing the same thing from line 79. Kindly check your code very well

Reply:
@adaugochi  line 80 returns a table with id , full_name and total_count
![image](https://user-images.githubusercontent.com/63324192/108372538-c8fc6180-71b3-11eb-85e4-ade41864c985.png)

while 75 returns the total number of passengers which is 9.
![image](https://user-images.githubusercontent.com/63324192/108372879-1d9fdc80-71b4-11eb-926a-b540162c1507.png)

Reply:
@adaugochi kindly review the PR and confirm with the screenshot for the out